src/tests/roc_pipeline/test_sender_sink.cpp

```diff
@@ -399,9 +399,21 @@ IGNORE_TEST(sender_sink, timestamp_mapping_remixing) {
     test::PacketReader packet_reader(arena, queue, format_map, packet_factory, dst_addr,
                                      PayloadType_Ch1);
 
+    core::nanoseconds_t cts = 0;
     for (size_t np = 0; np < ManyFrames / FramesPerPacket - 5; np++) {
-        packet_reader.read_nonzero_packet(SamplesPerPacket, packet_sample_spec,
-                                          unix_base);
+        packet::PacketPtr pp;
+        UNSIGNED_LONGS_EQUAL(status::StatusOK, queue.read(pp));
+        CHECK(pp);
+
+        if (np == 0) {
+            cts = pp->rtp()->capture_timestamp;
+            CHECK(cts >= unix_base);
```

Comment:
```suggestion
            CHECK(cts >= unix_base);
            CHECK(cts < unix_base + core::Millisecond);
```

(I tested, the check passes)